Improvements on CMake template (#5421)
cyberbotics/webots: docs/guide/using-your-ide.md

file(GLOB CPP_SOURCES *.cpp)
set(SOURCES ${C_SOURCES} ${CPP_SOURCES})

> # Set the  Webots home path (Changed it based on your installation method)

Reviewer: ```suggestion
# Set the  Webots home path (change it according to your installation method)
```

> set(WEBOTS_HOME "/snap/webots/current/usr/share/webots")
> #set(WEBOTS_HOME "/usr/share/webots")

Reviewer: I would prefer to document the standard installation path rather than the snap one:
```suggestion
set(WEBOTS_HOME "/usr/local/webots")
#set(WEBOTS_HOME "/snap/webots/current/usr/share/webots")
```

Author: Ok, I think most people prefer debs.
